frontend/src/app/components/team/team.component.ts

```diff
@@ -16,16 +19,22 @@ import { DialogService } from '../../services/dialog.service';
 export class TeamComponent implements OnInit {
   @Input({ required: true })
   public overviewEntity!: OverviewEntity;
+  addIconSrc: BehaviorSubject<string> = new BehaviorSubject('/assets/icons/new-icon.svg');
 
   constructor(
     private dialogService: DialogService,
     private refreshDataService: RefreshDataService,
-  ) {}
-
-  trackByObjectiveId: TrackByFunction<ObjectiveMin> = (index, objective) => objective.id;
+    private configService: ConfigService,
+  ) {
+    this.configService.config$.subscribe((config: ClientConfig) => {
+      this.addIconSrc.next(config.customStyles['okr-add-objective-icon']);
+    });
```

Comment:
```suggestion
    this.configService.config$.subscribe((config: ClientConfig) => {
      if(config.customStyles['okr-add-objective-icon']){
         this.addIconSrc.next(config.customStyles['okr-add-objective-icon']);
      }
    });
```
... and move it down here.

This has the benefit that there is only a single place where the fallback is set.